src/plots/cartesian/dragbox.js

```diff
@@ -1041,8 +1041,14 @@ function calcLinks(constraintGroups, xIDs, yIDs) {
 
 // still seems to be some confusion about onwheel vs onmousewheel...
 function attachWheelEventHandler(element, handler) {
-    if(element.onwheel !== undefined) element.onwheel = handler;
-    else if(element.onmousewheel !== undefined) element.onmousewheel = handler;
+    var wheelEventName = element.onwheel !== undefined ? 'wheel' : 'mousewheel';
+
+    if(element._onwheel) {
+        element.removeEventListener(wheelEventName, element._onwheel);
+    }
+    element._onwheel = handler;
+
+    element.addEventListener(wheelEventName, handler, {passive: false});
```

Comment:
Unfortunately, this might break older browsers, see:

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Safely_detecting_option_support

I couldn't find a list of _which_ older browsers does not support `{passive: true|false}`, but we should probably play it safe here.

Reply:
Good catch @etpinard - do we need a `has-passive` ala https://github.com/dfcreative/has-hover? Or even better, a module that would figure this out once and then export the correct 3rd arg to mark the listener active or passive:
```js
var eventOpts = require('event-opts');
element.addEventListener(wheelEventName, handler, eventOpts.active); // false or {passive: false}
element.addEventListener(wheelEventName, handler, eventOpts.passive); // false or {passive: true}
```

Reply:
There is a polyfill [default-passive-events](https://www.npmjs.com/package/default-passive-events) for that. We could use detection logic from there, I agree with @alexcjohnson.

Reply:
@etpinard @alexcjohnson https://github.com/plotly/plotly.js/pull/2251/commits/a32a21aac96e6efdeed5dffbcf193ecbf838a533 introduces `eventListenerOptionsSupported` method to detect proper way to attach events.